### PR TITLE
GUVNOR-2839: [Spreadsheet Decision Table] Date is not converted

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-2839

### CODE REVIEW

**Comment by manstis** [comment]
Part of an ensemble:

- https://github.com/kiegroup/drools-wb/pull/476
- https://github.com/kiegroup/kie-docs/pull/238

**Comment by manstis** [comment]
@Rikkola @karreiro Would you mind reviewing at your leisure?

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/DTCellValueUtilities.java [hunk 1]
```diff
@@ -116,172 +113,183 @@ public static void assertDTCellValue( final String type,
...
+                    callback.onConversionError(text,
+                                               dataType);
                 }
-                dcv.setNumericValue( shortValue );
+                dcv.setNumericValue(shortValue);
                 break;
         }
+    }
 
+    private static String stripQuotes(String text) {
```
Comment (karreiro) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/DTCellValueUtilities.java]: I think that this method deserves to be `protected` and tested.

**Comment by manstis** [comment]
@karreiro Your wish is my command :-)

**Pull request merged**